test/01-msg.js

```diff
@@ -4,7 +4,7 @@ const bfe = require('../')
 tape('01 msg type', function (t) {
   const values = [
     '%HZVnEzm0NgoSVfG0Hx4gMFbMMHhFvhJsG2zK/pijYII=.sha256', // classic
-    '%HZVnEzm0NgoSVfG0Hx4gMFbMMHhFvhJsG2zK/pijYII=.bbmsg-v1', // bendy-butt
+    'ssb:message/bendybutt-v1/HZVnEzm0NgoSVfG0Hx4gMFbMMHhFvhJsG2zK_pijYII=', // bendy-butt
```

Comment:
This is going to be a breaking channing in bfe.json?

Are we gonna commit to message or msg as the typeName (or allow both)?

Reply:
SSB URIs are `ssb:message/.....` and that's a thing already deployed in Patchfox, Manyverse, Patchfoo, so I wouldn't fiddle with those. Whether to support `ssb:msg/....` opens up the question of whether we should have aliases for URIs or not. Personally, I prefer when there is one way (no need to support two things, no need to make a choice which one to use at each case, etc), and anyway I think it's a bigger discussion outside of the scope of this PR

Reply:
> This is going to be a breaking channing in bfe.json?

Oh, and yes

Reply:
Yeah I'm fine with all that. Agree